Susi/Controllers/SkillDetailViewController/SkillDetailVCMethods.swift

```diff
@@ -479,5 +479,85 @@ extension SkillDetailViewController: UITableViewDelegate, UITableViewDataSource
             }
         }
     }
+    
+    //MARK:- Bookmark skill
+    
+    func showBookmarkButton() {
+        let delegate = UIApplication.shared.delegate as? AppDelegate
```

Comment:
define a variable `appDelegate` once and then use it instead of declaring it again and again.

Reply:
Done @chashmeetsingh